### PR TITLE
[2.13.x] DDF-4434 Changed the ID attribute on a LogoutRequest to comply with the SAML Specification

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do?
Changes the ID attribute on a LogoutRequest to comply with the SAML Specification

#### Who is reviewing it? 
@bakejeyner @Kjames5269 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@coyotesqrl

#### How should this be tested?
Run the [SAML Conformance Test Kit](https://github.com/codice/saml-conformance) and make sure it passes.

#### What are the relevant tickets?
[DDF-4434](https://codice.atlassian.net/browse/DDF-4434)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/core/security-core-services/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java [hunk 1]
```diff
@@ -135,8 +134,7 @@ public LogoutRequest extractSamlLogoutRequest(String samlLogoutResponse)
   @Override
   public LogoutRequest buildLogoutRequest(
       String nameIdString, String issuerOrEntityId, List<String> sessionIndexes) {
-    return buildLogoutRequest(
-        nameIdString, issuerOrEntityId, getUuidGenerator().generateUuid(), sessionIndexes);
+    return buildLogoutRequest(nameIdString, issuerOrEntityId, generateId(), sessionIndexes);
```
Comment (coyotesqrl) [inline platform/security/core/security-core-services/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java]: For other reviewers: it took me a minute to see what had actually changed; this is the line.

**Comment by blen-desta** [comment]
build now

**Comment by bakejeyner** [comment]
Hero ✔️

**Comment by blen-desta** [comment]
build now

**Pull request merged**